app/views/spree/products/_cart_form.html.erb

```diff
@@ -1,83 +1,165 @@
+          <%#= Spree.t(:add_to_cart) %>
+        <%# end %>
       </div>
-    <% end %>
+    <%# end %>
+  </div> -->
+<%# end %>
+<% is_product_available_in_currency = product_available_in_currency? %>
+<% default_variant = default_variant(@variants, @product) %>
+<div class="product-side-details d-flex flex-column justify-content-between h-100"  >
+  <div class="top-detail" >
+    <%= form_for :order, html: {
+      id: 'add-to-cart-form',
+      class: 'add-to-cart-form',
+      'data-product-summary': @product_summary.to_json,
+      'data-variants': product_variants_matrix(is_product_available_in_currency),
+      'data-variant-change-trigger-identifier': variant_change_identifier
+      } do |f| %>
+        <div class="product-options-header d-flex align-items-center justify-content-between" id="inside-product-cart-form" data-hook="inside_product_cart_form">
+          <div class="left">Choose your <strong>option:</strong></div>
+          <div class="right">
+            <div class="text-center text-md-left text-uppercase " id="product_variant_sku">
+                <%= default_variant.sku  %>
+            </div>
+          </div>
+        </div>
+        <ul class="size-availability" id="product-variants">
+          <% if @product.variants_and_option_values(current_currency).any? %>
+            <% @product.variants_and_option_values(current_currency).each_with_index do |variant, index| %>  
+              <li class="d-flex flex-wrap align-items-center justify-content-between  product-ov-js <%= index == 0 ? 'active' : '' %>"
+                data-varient-id="<%= variant.id %>" data-varient-sku="<%= variant.sku %>"> 
+              <% option_value = variant.size_options %>
+                <div class="left  add-to-cart-form-price price selling"  id="product-price"  data-hook="product_price">
+                <%= variant.price_in(current_currency).money %><span>/ $6.29 per oz.</span></div>
+                <div class="right"> 
+                  <%= option_value&.presentation %>
+                  <%= radio_button_tag "variant_id",  variant.id, index == 0 ? true : false,
+                          'data-price' => variant.price_in(current_currency).money,
+                          'data-in-stock' => variant.can_supply?,
+                          'data-backordered' => variant.backordered?,
+                          style: "display:none"
+                      %>
+            <% end%>
+              </div>
+          <% else %>
+            <%= hidden_field_tag "variant_id", @product.master.id %>
+              </li>
+          <% end %>
+        </ul>
+        <div class="d-flex justify-content-end mb-5">
+          <button type="button" class="btn btn-white expand-list">More Options</button>
+        </div>
+        <% if is_product_available_in_currency && @product.can_supply? %>
```

Comment:
This check shouldnt on quantity field. In default spree its only added on price. See snap of default spree as reference
<img width="942" alt="Screen Shot 2021-12-09 at 1 16 49 AM" src="https://user-images.githubusercontent.com/77246192/145277879-c6c1cb3a-6248-4bcc-a7c4-b87f4e7c7f5b.png">

Reply:
![Screenshot from 2021-12-09 13-56-33](https://user-images.githubusercontent.com/61042396/145365379-294c06a1-0d23-4e60-9008-ba53184e335e.png)
sir in default spree it is also applied on quantity kindly check it out